routes/routes.go

```diff
@@ -29,19 +27,15 @@ func Setup(ctx context.Context, r *mux.Router, cfg *config.Config, rend *render.
 	if cfg.MailHost == "localhost" {
 		auth = unencryptedAuth{auth}
 	}
-	mailAddr := fmt.Sprintf("%s:%s", cfg.MailHost, cfg.MailPort)
 
-	emailSender := email.SMTPSender{
-		Addr: mailAddr,
-		Auth: auth,
-	}
+	feedbackClient := sdk.New(cfg.FeedbackAPI.APIURL)
 
 	log.Info(ctx, "adding routes")
 	r.StrictSlash(true).Path("/health").HandlerFunc(hc.Handler)
 	r.StrictSlash(true).Path("/feedback").Methods("GET").HandlerFunc(handlers.GetFeedback(rend, cacheService))
-	r.StrictSlash(true).Path("/feedback").Methods("POST").HandlerFunc(handlers.AddFeedback(cfg.FeedbackTo, cfg.FeedbackFrom, false, rend, emailSender, cacheService))
+	r.StrictSlash(true).Path("/feedback").Methods("POST").HandlerFunc(handlers.AddFeedback(false, rend, cacheService, cfg, feedbackClient))
```

Comment:
I see `isPositive` is always false for both `/feedback` and `/feedback/tahnks`. This means we could just remove the parameter. But maybe it is there for some other reason that I am not aware of. Maybe it would be worth checking with someone with more experience with this frontend controller why this is the case.

Reply:
Didnt see this message until now. Will make the change.